src/System.Net.Http.WinHttpHandler/tests/FunctionalTests/WinHttpHandlerTest.cs

```diff
@@ -134,6 +134,47 @@ public void SendAsync_SlowServerRespondsAfterDefaultReceiveTimeout_ThrowsHttpReq
             }
         }
 
+        [OuterLoop]
+        [Theory]
+        /*
+            Take internationalized domain name 'Bangladesh' sample url from http://www.i18nguy.com/markup/idna-examples.html 
+            Saved in codeBase64 to avoid Visual Studio complains about encoding
+        */
+        [InlineData("aHR0cDovL+CmrOCmvuCmguCmsuCmvuCmpuCnh+Cmti5pY29tLm11c2V1bQ==", true, "icom.museum")]
```

Comment:
It isn't necessary to use Base64 encoding to create Unicode Uri literals.  Just use "\uuuu" notation in the literals.

i.e. "http://\u09AC\u09BE\u0982\u09B2\u09BE\u09A6\u09C7\u09B6.icom.museum"

Reply:
done